Source/VisualStudioTools/Private/VSTestAdapterCommandlet.cpp

```diff
@@ -138,40 +157,62 @@ int32 UVSTestAdapterCommandlet::RunTests( const FString& TestListFile, const FSt
 	return AllSuccessful ? 0 : 1;
 }
 
-
-void UVSTestAdapterCommandlet::ReadTestsFromFile( const FString& InFile, TArray< FAutomationTestInfo >& OutTestList )
+UVSTestAdapterCommandlet::UVSTestAdapterCommandlet()
 {
-	TSet< FString > TestCommands;
+	HelpDescription = TEXT("Commandlet for generating data used by Blueprint support in Visual Studio.");
+	HelpUsage = TEXT("<Editor-Cmd.exe> <path_to_uproject> -run=VSTestAdapter [-stdout -multiprocess -silent -unattended -AllowStdOutLogVerbosity -NoShaderCompile]");
+
+	HelpParamNames.Add(ListTestsParam);
+	HelpParamDescriptions.Add(TEXT("[Optional] The file path to write the test cases retrieved from FAutomationTestFramework"));
```

Comment:
Not critical, but these are not quite optional, they are parameter sets: if you have this one you cannot have the other two, but `RunTests` requires `TestResultsFile`, and if nothing is passed, we show Help and error out, correct?

You could make this simpler if you have always tree params `-run=VSTestAdapter -mode=[Run|List] -inputtests=[path] -output=[path]`, all required. 

Something to keep in mind for later if needed, but will need coordination on the VS side, so you want to time it correctly, maybe after the first version is out for testing.

Reply:
Will address in future PR soon.